test/e2e/runner/load.go

```diff
@@ -174,6 +188,43 @@ FOR_LOOP:
 	wg.Wait()
 }
 
+// loadProcess processes transactions by sending transactions received on the txCh
+// to the client.
+func loadProcessMultiple(ctx context.Context, txCh <-chan types.Tx, chSuccess chan<- struct{}, chFailed chan<- error, nodes []*e2e.Node, useInternalIP bool) {
+	clients := make(map[string]*rpchttp.HTTP, len(nodes))
+	var err error
+	s := struct{}{}
+
+	for _, n := range nodes {
+		if clients[n.Name] == nil {
+			if useInternalIP {
+				clients[n.Name], err = n.ClientInternalIP()
+			} else {
+				clients[n.Name], err = n.Client()
+			}
+			if err != nil {
+				logger.Info("non-fatal error creating node client", "error", err)
+				return
+			}
+		}
+	}
+	for tx := range txCh {
+		var wg sync.WaitGroup
+		wg.Add(len(nodes))
+		for _, n := range nodes {
+			go func(client *rpchttp.HTTP) {
```

Comment:
This will generate a lot of goroutines but we've seen that it works in practice.

Reply:
I am not sure how to do it differently so that the txs are sent in parallel.. We could use BroadcastTxAsync or have one go routine per X nodes (in case we have this situation as with 200 nodes where we send 100 duplicates).